apps/hash-graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -77,6 +81,10 @@ impl RoutedResource for EntityResource {
             "/entities",
             Router::new()
                 .route("/", post(create_entity::<S, A>).put(update_entity::<S, A>))
+                .route(
+                    "/:entity_id/public",
+                    post(make_entity_public::<A>).delete(make_entity_private::<A>),
+                )
```

Comment:
I wonder why we have the update route as `PUT /entities` rather than `PUT /entities/:entity_id` – these routes are arguably inconsistent now because the `entity_id` is in the body in the first but the path in the second. FWIW my expectation would be how you've done it for the new route

Reply:
I was wondering the exact same but didn't want to change this as a drive-by.

Reply:
Fair enough. Let's clean it up later then